### PR TITLE
Feature/mx 1539 get seq repo data from drop

### PR DESCRIPTION
**Pull request by rababerladuseladim**
# PR Context

# Added

# Changes

- get seq-repo data via mex-drop connector (was: file)

# Deprecated

# Removed

# Fixed

# Security

### CODE REVIEW

File: mex/seq_repo/extract.py [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
     Returns:
         Generator for Seq Repo resources
     """
-    settings = SeqRepoSettings.get()
-    with open(settings.default_json_file_path, encoding="utf-8") as file:
-        data = json.load(file)
-        for item in data:
-            yield SeqRepoSource.model_validate(item)
+    connector = DropApiConnector.get()
+    files = connector.list_files("seq_repo")
```
Comment (mr-kamran-ali) [inline mex/seq_repo/extract.py:20]: For me at this point: `files` is `{'entity-types': ['default']}`

which is a dict, resulting in `KeyError` on `data = connector.get_file("seq_repo", files[0])`

there is only one file in `drop` which I uploaded using mex-drop api itself. What am I missing?

**Review by mr-kamran-ali — COMMENTED** [review]
Code looks neat, I am having a tiny error for some reason.

File: mex/seq_repo/extract.py (hunk 1 above)
Comment (rababerladuseladim) [reply mex/seq_repo/extract.py]: Whoa, nice find. This was indeed broken and is fixed in e3d29750c947897b12bc1b16c6ba0b88fc9f33a3
For testing seq-repo locally, follow these steps:

1. in your local `mex-drop/data` directory, create a new folder `seq-repo`
2. copy the [default.json](https://raw.githubusercontent.com/robert-koch-institut/mex-extractors/main/assets/raw-data/seq-repo/default.json) from mex-extractors main branch into the seq-repo folder
3. in mex-drop: fire up mex-drop: `pdm run drop`
4. in mex-extractors: run seq-repo extractor: `pdm run seq-repo`

Does this work for you?

**Review by mr-kamran-ali — APPROVED** [review]
🚢

**Pull request merged**